styles/Canonical/025b-latinisms-to-reconsider.yml

```
@@ -0,0 +1,10 @@
+# 025 - Latinisms that we strongly advise against
+
+extends: existence
+message: Instead of 'a.m.', 'AM', 'p.m.', and 'PM', use 24-hour time.
+link: https://docs.ubuntu.com/styleguide/en#latin-words-and-phrases
+level: suggestion
+nonword: true
+tokens:
+  - '\b(?:a\.m\.(?!\w)|a.m(?![\.\w])|AM\b|A\.M\.(?!\w)|A\.M(?![\.\w]))'
+  - '\b(?:p\.m\.(?!\w)|p.m(?![\.\w])|PM\b|P\.M\.(?!\w)|P\.M(?![\.\w]))'
```

```suggestion
  - '\b(?:10|11|12|[1-9])(?:p\.m\.(?!\w)|PM\b|pm\b|P\.M\.(?!\w))'
```
Changed in accordance with the changes made to the AM rule.

---

Since `pm` doesn't have a homonym, the rule can be simpler: 451ff5a7fb495fbf320f3a91a999ecaddf84ed62